rfcs/20200624-pluggable-device-for-tensorflow/sample/eigen.BUILD

```diff
@@ -0,0 +1,78 @@
+# Description:
+#   Eigen is a C++ template library for linear algebra: vectors,
```

Comment:
This file and the folder `//sample/eigen3` seems to be redundant to the ones in `//sample/third_party`. Please delete them if they are indeed duplicates.

Reply:
done, removed it